### PR TITLE
Sd vertices

### PR DESCRIPTION
**Pull request by rpachaly**

### CODE REVIEW

File: flo2d/flo2d_ie/swmm_io.py
Comment (FLO-2DJJ) [inline flo2d/flo2d_ie/swmm_io.py:1]: 'ERROR 070618.1623' number repeated.

**Comment by FLO-2DJJ** [comment]
@rpachaly,

There is an error in _line 1078_ (a _try_ with a _except_ commented)

**Comment by rpachaly** [comment]
> @rpachaly,
> 
> There is an error in _line 1078_ of _swmm_io.py_ (a _try_ with a _except_ commented)

Is it an easy fix? I'm working on other stuff and I did not want to change gears now.

**Comment by FLO-2DJJ** [comment]
Seems easy to me. But it may have some other implications. Fix it when you
can.

Juan Jose Rodriguez
FLO-2D Software
www.flo-2d.com
***@***.***

File: flo2d/gui/storm_drain_editor_widget.py
[the remark this answers is not in the document]
Comment (FLO-2DKaren) [reply flo2d/gui/storm_drain_editor_widget.py:1]: @FLO-2DJJ Can you fix this so Robson can work on other stuff.  It should be an easy fix and I can test it.

**Comment by FLO-2DJJ** [comment]
@FLO-2DKaren, @rpachaly 

The mentioned error (try...except) fixed, and warning/errors numbers changed.

Please test it and merge if satisfied.

**Comment by rpachaly** [comment]
I just redid my testing here. All good. Only change is that I moved the curve message to the log.

**Pull request merged**